### PR TITLE
Feature: mixpanel events 

### PR DESCRIPTION
**Pull request by snova-jorgep**
- added basic mixed panel track events util
- included events in 
   - multimodal enterprise knowledge retriever
   - search assitant
   - enterprise knowledge retriever
   - sambanova scribe
   - function calling
   - benchmarking
   - financial assistant

### CODE REVIEW

**Review by snova-rodrigom — APPROVED** [review]
LGTM! thanks @snova-jorgep . Tested benchmarking kit functionalities and look good.

File: financial_assistant/streamlit/utilities_app.py [hunk 1]
```diff
@@ -45,6 +56,19 @@ def initialize_session(
     if 'launch_time' not in session_state:
         session_state.launch_time = datetime.datetime.now()
 
+    # Mixpanel events
+    if 'st_session_id' not in session_state:
+        session_state.st_session_id = str(uuid.uuid4())
```
Comment (snova-francescar) [inline financial_assistant/streamlit/utilities_app.py:61]: We already have a SESSION_ID randomly generated in `constants.py`. Could you please replace this with
```
if 'st_session_id' not in session_state:
        session_state.st_session_id = SESSION_ID
```

**Review by snova-francescar — APPROVED** [review]
I reviewed the `financial_assistant` changes, and I only have one comment to address.

File: financial_assistant/streamlit/utilities_app.py (hunk 1 above)
Comment (snova-jorgep) [reply financial_assistant/streamlit/utilities_app.py]: Sure, Replaced

**Pull request merged**